src/app/store/slices/sharedLinks/index.ts

```diff
@@ -100,7 +101,7 @@ const getSharedLinkThunk = createAsyncThunk<string | void, GetLinkPayload, { sta
       const share = await shareService.createShare(requestPayload);
       const link = shareService.getLinkFromShare(share, code, mnemonic, requestPayload.type);
       navigator.clipboard.writeText(link);
-      notificationsService.show({ text: 'Share link copied to clipboard', type: ToastType.Success });
+      notificationsService.show({ text: t('notificationMessages.copyLink'), type: ToastType.Success });
```

Comment:
If `t `of i`mport { t } from 'i18next';`  works as expected maybe could use it in some functions instead of pass the translation function as prop.